### PR TITLE
feat(verification): remove sanctioned countries from phone dropdown

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Removes sanctioned countries from the phone verification dropdown https://app.zenhub.com/workspaces/acquisition-squad-sprint-board-6010683afabec1001a090887/issues/valora-inc/wallet/2243

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

Added new unit tests and manually verified.

### How others should test

* Be not phone verified
* Navigate to the Phone Verification screen
* Search for 'Syria' or 'Iran' in the phone country code dropdown

expected: Neither country shows up as an option in the dropdown

### Related issues

- https://app.zenhub.com/workspaces/acquisition-squad-sprint-board-6010683afabec1001a090887/issues/valora-inc/wallet/2243

### CODE REVIEW

File: packages/mobile/src/account/selectors.ts [hunk 1]
```diff
@@ -19,7 +23,9 @@ export const e164NumberSelector = (state: RootState) => state.account.e164PhoneN
 export const pictureSelector = (state: RootState) => state.account.pictureUri
 export const defaultCountryCodeSelector = createSelector(
   (state: RootState) => state.account.defaultCountryCode,
-  (defaultCountryCode) => defaultCountryCode || inferCountryCode()
+  (defaultCountryCode) => {
+    return defaultCountryCode || inferCountryCode()
+  }
```
Comment (satish-ravi) [inline packages/mobile/src/account/selectors.ts:28]: just curious, is this from a linter/formatter?

**Review by satish-ravi — APPROVED** [review]

**Review by jeanregisser — COMMENTED** [review]
Sanctioned countries were already not able to do phone number verification.
They were greeted with an error message when trying to do so:
https://github.com/valora-inc/wallet/blob/029b7f6bdda17fabb4a7aae9b349f6c1e9730e38/packages/mobile/src/verify/VerificationEducationScreen.tsx#L219-L223

IMHO, this change provides a worse experience for users of sanctioned countries, they won't understand why their country is not listed.

Can we reconsider this change?

**Comment by cajubelt** [comment]
Should the full list of blocked countries be in the test steps of the description? seeing these in the GH issue:
Cuba 11
North Korea 0
Iran 115
Syria 5

File: packages/mobile/src/onboarding/registration/SelectCountry.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <Provider store={createMockStore({})}>
+        <SelectCountry {...mockScreenProps} />
+      </Provider>
+    )
+    // Germany shows up
+    fireEvent.changeText(tree.getByTestId('SearchInput'), 'Germany')
+    expect(tree.queryByText('Germany')).not.toBeNull()
+    // Syria does not
+    fireEvent.changeText(tree.getByTestId('SearchInput'), 'Syria')
+    expect(tree.queryByText('Syria')).toBeNull()
```
Comment (cajubelt) [inline packages/mobile/src/onboarding/registration/SelectCountry.test.tsx:28]: can you add checks for the other 3 countries too?

**Review by cajubelt — APPROVED** [review]
nice, didn't know we already had the list of sanctioned countries

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Test Flight Release build V1.31.0 and updated status in test rails.

- User is not able to see Syria and Iran country name in phone country drop down

**Test rail links:**
**iOS15:** https://valoraapp.testrail.io/index.php?/tests/view/30604

**Devices:** iPhone 13 mini(15.1.1)

**CC:** @jh2oman 
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT , We have verified above task on Latest Android internal release build and updated status in test rails.

Observation : User is not able to see Syria and Iran country name in phone country drop down

Test rail links:
Android 11: https://valoraapp.testrail.io/index.php?/tests/view/30697
Devices: Google Pixel 2XL(11.0)

CC: @jh2oman 
Thanks..!!

**Pull request merged**